tap_github/tap.py

```diff
@@ -83,6 +83,19 @@ def logger(cls) -> logging.Logger:
         th.Property("repositories", th.ArrayType(th.StringType)),
         th.Property("user_usernames", th.ArrayType(th.StringType)),
         th.Property("user_ids", th.ArrayType(th.StringType)),
+        th.Property(
+            "milestones",
+            th.ObjectType(
+                th.Property(
+                    "state",
+                    th.StringType,
+                    description=(
+                        "Configures which states are of interest. "
+                        "Must be one of [open, closed, all], defaults to open."
+                    ),
+                ),
+            ),
```

Comment:
Nit: add a little more metadata to the new settings

```suggestion
            th.ObjectType(
                th.Property(
                    "state",
                    th.StringType,
                    description=(
                        "Configures which states are of interest. "
                        "Must be one of [open, closed, all], defaults to open."
                    ),
                    default="open",
                    allowed_values=["open", "closed", "all"],
                ),
            ),
            descriptions="Setting specific to the 'milestones' stream."
```

Reply:
Nice!